assets/xml/overlays/ovl_Oceff_Spot.xml

```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Spot" BaseAddress="0x80972680" RangeStart="0x720" RangeEnd="0xDF8">
+        <Texture Name="sTex" OutName="sun_song_effect" Format="i8" Width="32" Height="32" Offset="0x720"/>
+        <Array Name="sCylinderVtx" Count="27" Offset="0xB20">
+            <Vtx/>
+        </Array>
+        <DList Name="sCylinderMaterialDL" Offset="0xCD0"/>
+        <DList Name="sCylinderModelDL" Offset="0xD68"/>
```

Comment:
```suggestion
        <Array Name="sSunSongEffectCylinderVtx" Count="27" Offset="0xB20">
            <Vtx/>
        </Array>
        <DList Name="sSunSongEffectCylinderMaterialDL" Offset="0xCD0"/>
        <DList Name="sSunSongEffectCylinderModelDL" Offset="0xD68"/>
```
These should have the actor name or the English name prefixed to them

Reply:
Seems to apply to a lot of the other ocarina song effect PRs you've got, since these files are so similar. I'll give you a chance to update those, and I'll check in and comment on those PRs later